modules/react-native-status/nodejs/status.cpp

```diff
@@ -761,6 +761,26 @@ void _Logout(const FunctionCallbackInfo<Value>& args) {
 
 }
 
+void _AcceptTerms(const FunctionCallbackInfo<Value>& args) {
+	Isolate* isolate = args.GetIsolate();
+
+	if (args.Length() != 0) {
+		// Throw an Error that is passed back to JavaScript
+		isolate->ThrowException(Exception::TypeError(
+			String::NewFromUtf8Literal(isolate, "Wrong number of arguments for AcceptTerms")));
+		return;
+	}
+
+	// Check the argument types
+
+	// Call exported Go function, which returns a C string
+	char *c = AcceptTerms();
+
+	Local<String> ret = String::NewFromUtf8(isolate, c).ToLocalChecked();
+	args.GetReturnValue().Set(ret);
+	delete c;
+}
+
```

Comment:
So the code here looks good, but I'm wondering if we have to create a binding in the CPP code. I only ask because I've noticed that there are some other OBJ-C and Kotlin bindings that don't have their own CPP bindings.

For example:
https://github.com/status-im/status-mobile/blob/1cb25cf61aa51d3f0a594b5d18cb8d26205d0048/modules/react-native-status/android/src/main/java/im/status/ethereum/module/AccountManager.kt#L317-L328

So I'm wondering if we need this code, and/or if we're missing some other code.

Reply:
Does someone have more context on this?
cc @siddarthkay @smohamedjavid

Reply:
@seanstrom : we only need cpp bindings for stuff we use in integration tests.